ttnn/cpp/ttnn/operations/conv2d.cpp

```diff
@@ -668,7 +668,7 @@ std::tuple<ttnn::Tensor, uint32_t, uint32_t, ttnn::Tensor, std::optional<ttnn::T
         Tensor matmul_input = input_tensor_post_tm;
         if (stride[0] > 1) {
             // run downsample
-            matmul_input = tt::tt_metal::downsample(
+            matmul_input = ttnn::operations::data_movement::downsample(
```

Comment:
It should not be in data_movement namespace. I think it belongs to convolutions, right?

Reply:
@ayerofieiev-tt  upsample is part of data_movement. so shall I keep it in data_movement or you want me to move it to conv only??

Reply:
Does upsample really semantically belong to data_movement or its just that someone put it into that namespace? :)

Reply:
kept in data_movement as of now.